test/test.js

```diff
@@ -136,17 +152,50 @@ test('Logs', async t => {
   })
 
   await t.test('Follow', async t => {
-    for (const flag of ['-f', '--follow']) {
+    await t.test('Read logs', async t => {
+      for (const flag of ['-f', '--follow']) {
```

Comment:
In my experience, it's better to move this for-loop outside of the test case, so that we have a new test for each variant we are checking. Such setup makes it easier to troubleshoot test failures, as they are more isolated.

```js
for (const flag of ['-f', '--follow']) {
  await t.test(`Read logs with "${flag}"`, async t => {
    // etc.
  })
}
```

Reply:
This is a great suggestion! I knew something was off :D